src/runtime/scheduler_avr.S

```diff
@@ -187,3 +187,55 @@ tinygo_switchToScheduler:
 
     // Return into the scheduler, as if tinygo_switchToTask was a regular call.
     ret
+
+.global tinygo_localscan
+.type tinygo_localscan, %function
+tinygo_localscan:
+    // Save callee-saved registers.
+    push r29 // Y
+	  push r28 // Y
```

Comment:
This file contains some inconsistent whitespace.

(While I personally prefer spaces over tabs, it is far more important to be consistent with the surrounding code).